index.js

```diff
@@ -226,6 +227,23 @@ module.exports = {
     importer.import('vendor/lunr/lunr.js', {
       using: [{ transformation: 'amd', as: 'lunr' }],
     });
+
+    if (EmberApp.env() === 'production') {
```

Comment:
Should we add some tests to ensure the behavior on production/development environment?